python/IAM_USER_PERMISSION_BOUNDARY_CHECK/IAM_USER_PERMISSION_BOUNDARY_CHECK.py

```diff
@@ -0,0 +1,489 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+"""
+#####################################
+## Gherkin ##
+#####################################
+
+Rule Name:
+IAM_USER_PERMISSION_BOUNDARY_CHECK
+
+Description:
+Check if all the IAM users have Permission boundary attached. The rule is NON_COMPLAINT if the PermissionBoundary is not attached to the IAM user.
+
+Trigger:
+Periodic
+
+Reports on:
+AWS::IAM::User
+
+Rule Parameters:
+policyArns
+(Optional) 

```

Comment:
remove the new line at the end of this line. To make a single line with `(Optional) Comma-separated list of permission boundary policy ARNs, that are expected to be attached to the IAM Users`